README.md

```diff
@@ -62,6 +62,14 @@ See https://github.com/ddclient/ddclient/releases
 
 ## INSTALLATION
 
+<img align="right" src="https://repology.org/badge/vertical-allrepos/ddclient.svg">
+
+### Distribution Package
+
+The easiest way to install ddclient is to install a package offered by your operating system. Packages are available for the following distributions:
```

Comment:
```suggestion
### Distribution Package

<img align="right" src="https://repology.org/badge/vertical-allrepos/ddclient.svg">
The easiest way to install ddclient is to install a package offered by your
operating system. See the image to the right for a list of distributions with a
ddclient package.
```

Reply:
I don't know about this. It kinda looked better when it was right under the installation headline and we didn't waste some whitespace. The badge is just to big tbh.